satellite/satellitedb/dbx/satellitedb.dbx.go

```diff
@@ -485,7 +484,7 @@ CREATE TABLE buckets (
 	project_id bytea NOT NULL REFERENCES projects( id ),
 	name bytea NOT NULL,
 	path_cipher integer NOT NULL,
-	created_at timestamp NOT NULL,
+	created_at timestamp with time zone NOT NULL,
```

Comment:
Why is this change?

Reply:
This must have happened on the go-generate because I didn't change this manually.